roscomm/intlist[] current_selection